koku/cost_models/serializers.py

```diff
@@ -451,6 +492,9 @@ def validate(self, data):
         if not data.get("currency"):
             data["currency"] = get_currency(self.context.get("request"))
 
+        if not data.get("distribution_info"):
+            data["distribution_info"] = self.validate_distribution_json(data)
+
```

Comment:
If we aren't requiring a `distribution_info` to be sent with the payload, why are we creating one?